Mathlib/FieldTheory/SeparableClosure.lean

```diff
@@ -176,6 +181,7 @@ theorem separableClosure.normalClosure_eq_self :
 
 /-- If `E` is normal over `F`, then the separable closure of `F` in `E` is Galois (i.e.
 normal and separable) over `F`. -/
+@[stacks 0EHK]
```

Comment:
I don't think this tag is right.

Reply:
Thanks. This is typo. This should be `0EXK`.

Reply:
Fixed.